.circleci/config.yml

```diff
@@ -28,6 +28,7 @@ commands:
 
     install-carthage:
         steps:
+            - macos/install-rosetta
```

Comment:
The same step was added to the `install-carthage` command.
Does it make sense to add a separate command to install-rosetta